www/src/data/sidebars/doc-links.yaml

```diff
@@ -93,26 +93,32 @@
               breadcrumbTitle: ESLint
             - title: Proxying API Requests
               link: /docs/api-proxy/
-        - title: Images, Files, and Video
+        - title: Images, Files & Video
           link: /docs/images-and-files/
           items:
-            - title: Importing Assets Directly Into Files
+            - title: Importing Assets Directly into Files
               link: /docs/importing-assets-into-files/
-              breadcrumbTitle: Importing Assets Into Files
+              breadcrumbTitle: Importing Assets into Files
             - title: Using the Static Folder
               link: /docs/static-folder/
+              breadcrumbTitle: Static Folder
             - title: Using gatsby-image
               link: /docs/using-gatsby-image/
+              breadcrumbTitle: gatsby-image
```

Comment:
Using the default title "Using gatsby-image" makes sense here in my opinion as gatsby-image is a bit bland.

```suggestion
```

Reply:
I was conflicted on this as well but there are other breadcrumb titles that are pretty short and bland, so I thought we should follow suit here. For example, "Customizing html.js" under the "Custom Configuration" section has a breadcrumb title of "html.js"

Reply:
To chime in here, the reason for omitting words is to make them less redundant. “Html.js” in the case of the breadcrumb comes after “Custom configuration”, so the phrases work together. Hopefully that can act as a guide for how to work with breadcrumb titles.

Reply:
Thanks for chiming in! That makes a lot more sense when you consider the entire breadcrumb chain. I'll revert it back to using the default title then as @wardpeet suggested.